docs/tutorial/rln-chat2-live-testnet.md

```diff
@@ -0,0 +1,125 @@
+# Building a local network of spam-protected chat2 clients 
+
+This document is a tutorial on how to run chat2 in spam-protected/rate-limited mode on a designated content topic  `/toy-chat/2/luzhou/proto` using the waku-RLN-Relay protocol.
+You will connect your chat2 client to waku2 test fleets.
+Note that test fleets will not filter spam messages, they merely route messages.
+Spam detection takes place at the chat2 users.
+In this setting, you should try to spam the network by violating the message rate limit i.e.,
+sending more than one message per epoch. 
+At the time of this tutorial, the epoch duration is set to `10` seconds.
+You can inspect the current epoch value by checking the following [constant variable](https://github.com/status-im/nim-waku/blob/21cac6d491a6d995a7a8ba84c85fecc7817b3d8b/waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim#L119) in the nim-waku codebase.
+Your messages will be routed via test fleets and will arrive at other live chat2 clients that are running in rate-limited mode over the same content topic i.e., `/toy-chat/2/luzhou/proto`.
+Your samp activity will be detected by them and a proper message will be shown on their console.  
+
+# Set up
+## Build chat2
+First, build chat2 with the RLN flag set to true.
+
+```
+make chat2 RLN=true
+```
+
+## Setup a chat2 node in rate-limited mode
+Run the following command to set up your chat2 client. 
+
+```
+./build/chat2 --content-topic:/toy-chat/2/luzhou/proto --ports-shift=1 --fleet:test --rln-relay:true --rln-relay-membership-index:your_index
```

Comment:
For Bob you set up `your_index=2` as default, it is not possible to do the same here, e.g. 1?
Although you explain it later, people that would like to quickly test the feature by copy/pasting would receive an error.

Reply:
The `your_index` should be replaced with the actual numerical index. 

> people that would just quickly test this feature by copy/pasting commands will receive an error.

Can you please explain what kind of error? even by replacing a numerical index for `your_index`?

Reply:
I meant the error that the string "your_index" cannot be interpreted. There is nothing wrong with the tutorial, since it is specified later that it has to be replaced with an integer, but a quick reader (i.e., who does not notice that `your_index` has to be replaced and only copies/pastes commands to quickly test them), will receive such error. 
While this is not the case for Bob, since for him the index is already set to 2, I wondered if it makes sense to fix Alice's index to 1 just in the command example and clearly keep the specification that in real cases it should be changed to `your_index`.